packages/contract-tests/scripts/start-mock-bff.sh

```
@@ -0,0 +1,114 @@
+fi
+
+BFF_LOG_FILE="${BFF_DIR}/bff-mock.log"
+log_info "Starting Mock BFF server on port ${PORT}..."
+./bff-mock --mock-k8s-client --mock-mr-client --port "$PORT" > "$BFF_LOG_FILE" 2>&1 &
+BFF_PID=$!
+echo "$BFF_PID" > "$BFF_DIR/bff.pid"
+log_success "Mock BFF started (PID: $BFF_PID) logs: $BFF_LOG_FILE"
+popd >/dev/null
+
+# Wait for readiness
+for i in $(seq 1 30); do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)